vsce/package.json

```diff
@@ -303,7 +310,8 @@
     "scripts": {
         "clean": "rimraf client/out && rimraf server/out",
         "compile": "tsc -b",
-        "watch": "tsc -b -w",
+        "cc": "npm run clean && npm run compile",
```

Comment:
Was this included by mistake?

Reply:
It helped me while testing this change. I can include it in a separate pull request if you want.

Reply:
No biggie.